docs/docs/guides/custom-locations.md

```diff
@@ -40,13 +40,12 @@ docker compose up -d
 ```
 
 :::note
-Because of the underlying properties of docker bind mounts, it is not recommended to mount the `upload/` and `library/` folders as separate bind mounts if they are on the same device.
-For this reason, we mount the HDD or network storage to `/usr/src/app/upload` and then mount the folders we want quick access to below this folder.
+Due to the underlying properties of docker bind mounts, it is not recommended to mount the `upload/` and `library/` folders as separate bind mounts if they are on the same device.
+For this reason, we mount the hard drive (HDD) or the network storage (NAS) to `/usr/src/app/upload` and then mount the folders we want to access under that folder.
```

Comment:
"the HDD or network storage" is fine IMO. No need to explain those concepts :P

Reply:
Sure! I will amend the commit later this weekend

Reply:
Just came across this; “because of” is actually better there because it describes a verb not a noun. If you don’t like how it sounds then you can swap it for “owing to”, which is synonymous.

Reply:
Modification done, see comment below